ActionSheet: Update demo description (#2136)
DevExpress/devextreme-demos: JSDemos/Demos/ActionSheet/Basics/description.md

> The ActionSheet component is a popup sheet that contains a set of buttons. These buttons allow users to perform custom actions (call, send, delete, edit, etc). This demo demonstrates how to create a simple ActionSheet and handle the item clicks.

Reviewer: - When you use "pop-up" as an adjective, always hyphenate it.
- "demo demonstrates" - avoid repetition.
- "handle the item clicks" - ActionSheet has a specific type of items, so there's no reason to use a broader term.

```suggestion
The ActionSheet component is a pop-up sheet that contains a set of buttons. These buttons allow users to perform custom actions related to a single task (call, send a message, delete, or edit a selected contact). This demo shows how to create a simple ActionSheet and handle button clicks.
```

> ## Specify Controls and Title

Reviewer: Again, ActionSheet has a specific type of controls.

```suggestion
## Specify Buttons and Title
```

ActionSheet can contain multiple buttons. Use either the [dataSource](/Documentation/ApiReference/UI_Components/dxActionSheet/Configuration/#dataSource) or [items](/Documentation/ApiReference/UI_Components/dxActionSheet/Configuration/items/) properties to add buttons and configure them. For each button you can specify a [text](/Documentation/ApiReference/UI_Components/dxActionSheet/Configuration/items/#text), [type](/Documentation/ApiReference/UI_Components/dxActionSheet/Configuration/items/#type), [icon](/Documentation/ApiReference/UI_Components/dxActionSheet/Configuration/items/#icon), [template](/Documentation/ApiReference/UI_Components/dxActionSheet/Configuration/items/#template) and other properties. In this demo, the ActionSheet buttons are configured in a simple array assigned to the **dataSource** property. Each button includes the **text** property only.

If you want to display a title, assign the needed text to the [title](/Documentation/ApiReference/UI_Components/dxActionSheet/Configuration/#title) property.

> ## Handle the Item Clicks

Reviewer: ```suggestion
## Handle Button Clicks
```

> The component can perform a custom action when the user clicks the button. Use the [onItemClick](/Documentation/ApiReference/UI_Components/dxActionSheet/Configuration/#onItemClick) property to handle a custom item click.
> 
> You can also handle the click on the default **Cancel** button. For this, assign a custom handler to the [onCancelClick](/Documentation/ApiReference/UI_Components/dxActionSheet/Configuration/#onCancelClick) property. 
> 
> In this demo, the ActionSheet component displays notification with a button name, when you click the button. 
> 
> 

Reviewer: - "The component can perform a custom action when the user clicks the button." - We already told about it in the intro.
- When there are many short paragraphs, text looks choppy. Strive to keep the balance.

```suggestion
You can assign a function to the [onItemClick](/Documentation/ApiReference/UI_Components/dxActionSheet/Configuration/#onItemClick) property to handle button clicks. Use the `e.itemData` field within the function to determine which button was clicked. You can also use the [onCancelClick](/Documentation/ApiReference/UI_Components/dxActionSheet/Configuration/#onCancelClick) property handle a click on the built-in **Cancel** button. In this demo, the ActionSheet component displays a notification with a button name when you click a button. 
```

The ActionSheet component is a pop-up sheet that contains a set of buttons. These buttons allow users to perform custom actions related to a single task (call, send a message, delete, or edit a selected contact). This demo shows how to create a simple ActionSheet and handle button clicks.

## Specify Buttons and Title

ActionSheet can contain multiple buttons. To specify buttons and configure them, use one of these properties:

> - [items[]](/Documentation/ApiReference/UI_Components/dxActionSheet/Configuration/items/) 

Reviewer: ```suggestion
ActionSheet can contain multiple buttons. To specify and configure them, use one of these properties:

- [items[]](/Documentation/ApiReference/UI_Components/dxActionSheet/Configuration/items/)     
```

Accepts a local data array.

> - [dataSource](/Documentation/ApiReference/UI_Components/dxActionSheet/Configuration/#dataSource)

Reviewer: Four spaces create a real line break. Without them, it's all written in one line.

```suggestion
- [dataSource](/Documentation/ApiReference/UI_Components/dxActionSheet/Configuration/#dataSource)     
```

Accepts a local data array or a [DataSource](/Documentation/ApiReference/Data_Layer/DataSource/) object. This object works with local and remote arrays and allows you to shape data. 

> For each button you can specify a [text](/Documentation/ApiReference/UI_Components/dxActionSheet/Configuration/items/#text), [type](/Documentation/ApiReference/UI_Components/dxActionSheet/Configuration/items/#type), [icon](/Documentation/ApiReference/UI_Components/dxActionSheet/Configuration/items/#icon), [template](/Documentation/ApiReference/UI_Components/dxActionSheet/Configuration/items/#template) and other properties. In this demo, the ActionSheet buttons are configured in a simple array assigned to the **dataSource** property. Each button includes the **text** property only.

Reviewer: ```suggestion
For each button, you can specify a [text](/Documentation/ApiReference/UI_Components/dxActionSheet/Configuration/items/#text), [type](/Documentation/ApiReference/UI_Components/dxActionSheet/Configuration/items/#type), [icon](/Documentation/ApiReference/UI_Components/dxActionSheet/Configuration/items/#icon), [template](/Documentation/ApiReference/UI_Components/dxActionSheet/Configuration/items/#template), and other properties. In this demo, the ActionSheet buttons are configured in a local array assigned to the **dataSource** property. Each button includes the **text** property only.
```

## Specify Buttons and Title

ActionSheet can contain multiple buttons. To specify and configure them, use one of these properties:

- [items[]](/Documentation/ApiReference/UI_Components/dxActionSheet/Configuration/items/)     
Accepts a local data array.

- [dataSource](/Documentation/ApiReference/UI_Components/dxActionSheet/Configuration/#dataSource)     
Accepts a local data array or a [DataSource](/Documentation/ApiReference/Data_Layer/DataSource/) object. This object works with local and remote arrays and allows you to shape data. 

For each button, you can specify a [text](/Documentation/ApiReference/UI_Components/dxActionSheet/Configuration/items/#text), [type](/Documentation/ApiReference/UI_Components/dxActionSheet/Configuration/items/#type), [icon](/Documentation/ApiReference/UI_Components/dxActionSheet/Configuration/items/#icon), [template](/Documentation/ApiReference/UI_Components/dxActionSheet/Configuration/items/#template), and other properties. In this demo, the ActionSheet buttons are configured in a local array assigned to the **dataSource** property. Each button includes the **text** property only.

> If you want to display a title, assign the needed text to the [title](/Documentation/ApiReference/UI_Components/dxActionSheet/Configuration/#title) property.

Reviewer 2: ```suggestion
If you want to display a title, assign the text to the [title](/Documentation/ApiReference/UI_Components/dxActionSheet/Configuration/#title) property.
```

## Handle Button Clicks

> You can assign a function to the [onItemClick](/Documentation/ApiReference/UI_Components/dxActionSheet/Configuration/#onItemClick) property to handle button clicks. Use the `e.itemData` field within the function to determine which button was clicked. You can also use the [onCancelClick](/Documentation/ApiReference/UI_Components/dxActionSheet/Configuration/#onCancelClick) property handle a click on the built-in **Cancel** button. In this demo, the ActionSheet component displays a notification with a button name when you click a button. 

Reviewer 2: ```suggestion
You can assign a function to the [onItemClick](/Documentation/ApiReference/UI_Components/dxActionSheet/Configuration/#onItemClick) property to handle button clicks. Use the `e.itemData` field within the function to determine which button was clicked. You can also use the [onCancelClick](/Documentation/ApiReference/UI_Components/dxActionSheet/Configuration/#onCancelClick) property handle a click on the built-in **Cancel** button. In this demo, the ActionSheet component displays a notification with the button's name when you click a button. 
```
